packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -544,14 +545,15 @@ export default class Meeting extends StatelessWebexPlugin {
   resourceUrl: string;
   selfId: string;
   state: any;
-  localAudioTrackMuteStateHandler: (event: TrackMuteEvent) => void;
-  localVideoTrackMuteStateHandler: (event: TrackMuteEvent) => void;
-  underlyingLocalTrackChangeHandler: () => void;
+  localAudioStreamMuteStateHandler: (muted: boolean) => void;
+  localVideoStreamMuteStateHandler: (muted: boolean) => void;
+  localOutputTrackChangeHandler: () => void;
   roles: any[];
   environment: string;
   namespace = MEETINGS;
   annotationInfo: AnnotationInfo;
   allowMediaInLobby: boolean;
+  sendSlotManager: SendSlotManager = new SendSlotManager(LoggerProxy);
```

Comment:
I don't think we need to expose the sendSlotManager to the app, so this should probably be private

Reply:
Done in #3053